### PR TITLE
[Site] Change terminal output for k8s platform

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <noreply@github.com>

**Description**

NOTE: Sign off may differ in commits because I used GitHub CodeSpaces to work with this repo (due to >750MB) size

This PR is related to this issue [meshery/meshery#4233](https://github.com/meshery/meshery/issues/4233)

**Notes for Reviewers**
Still WIP

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Meshery/Meshery-terminal/index.js [hunk 1]
```diff
@@ -155,135 +155,55 @@ const MesheryTerminal = () => {
...
                     code: [
-                      "Removing installation files and starting Meshery",
-                      "Removing installation files and starting Meshery.",
-                      "Removing installation files and starting Meshery..",
-                      "Removing installation files and starting Meshery...",
-                      "Removing installation files and starting Meshery....",
-                      "Removing installation files and starting Meshery.....",
-                      "Removing installation files and starting Meshery......",
-                      "Removing installation files and starting Meshery.......",
+                      "downloading manifest files from https://github.com/meshery/meshery/tree/v0.5.60/install/deployment_yamls/k8s",
```
Comment (warunicorn19) [inline src/sections/Meshery/Meshery-terminal/index.js:158]:
```suggestion
                      "downloading manifest files from https://github.com/meshery/meshery/tree/v0.6.0/install/deployment_yamls/k8s",
```
Considering the fact we're switching to k8's from v6, let's keep it v0.6.0 here

**Comment by l5io** [comment]
🚀 Preview for commit cf7eb3ea617e5d481500cc01875ca85bf5ab8000 at: https://615c7bb0590a6a083f1541e1--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9a0cce49be0eb7ef843332757798e6be3524bca1 at: https://615d0390d67e6c4473b5d565--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 414c14dbb13af03427f9c420bfd3de90f7285ecc at: https://615d2ba00daf1d6579cc18fa--layer5ng.netlify.app

**Comment by alphaX86** [comment]
Oh I didn't notice that..... Could you provide suggestions for the frame limits in this case?

**Comment by l5io** [comment]
🚀 Preview for commit cd0e942922d1ef6d25be3f4ddbced054a43eb7f5 at: https://615dc7e83c862150d544a2fa--layer5ng.netlify.app

**Pull request merged**